nautobot_chatops_ipfabric/worker.py

```diff
@@ -832,15 +852,15 @@ def get_wireless_clients(dispatcher, ssid=None, snapshot_id=None):
         ],
         [
             (
-                client["controller"],
-                client["siteName"],
-                client["apName"],
-                client["client"],
-                client["clientIp"],
-                client["ssid"],
-                client["rssi"],
-                client["signalToNoiseRatio"],
-                client["state"],
+                client[IpFabric.WIRELESS_CLIENT_COLUMNS[0]],
+                client[IpFabric.WIRELESS_CLIENT_COLUMNS[1]],
+                client[IpFabric.WIRELESS_CLIENT_COLUMNS[2]],
+                client[IpFabric.WIRELESS_CLIENT_COLUMNS[3]],
+                client[IpFabric.WIRELESS_CLIENT_COLUMNS[4]],
+                client[IpFabric.WIRELESS_CLIENT_COLUMNS[5]],
+                client[IpFabric.WIRELESS_CLIENT_COLUMNS[6]],
+                client[IpFabric.WIRELESS_CLIENT_COLUMNS[7]],
+                client[IpFabric.WIRELESS_CLIENT_COLUMNS[8]],
```

Comment:
just. a suggestion, to cleaner code, I would go with tuple comprehension like

(client[IpFabric.WIRELESS_CLIENT_COLUMNS[i]] for i in range(len(IpFabric.WIRELESS_CLIENT_COLUMNS))

Reply:
Nice one!

Reply:
Turns out there is no tuple comprehension. That syntax is reserved for generator objects. Just used list comprehension instead.